tests/core/test_llvm_intrinsics.out

```diff
@@ -62,12 +62,12 @@ llvm_floor_f64 -2.0
 llvm_round_f64 20.0
 llvm_round_f64 21.0
 llvm_round_f64 42.0
-llvm_round_f64 -20.0
+llvm_round_f64 -21.0
```

Comment:
i'm a little worried about changing rounding semantics as part of this merge. optimally we'd do it separately.

I think I remember you having good reasons for this change, where are they documented?

Reply:
The implementation of `round` was incorrect,  `round*()` and `llvm_round_f*` are specced to always round away from zero. They don't have a controllable rounding mode. i.e. against the specification: https://github.com/kripken/emscripten/issues/5716 and https://github.com/kripken/emscripten/pull/5400/commits/7036e58dc7f2038d398cb02da85b6772ff45e3df.

Reply:
Ok, sounds good. But please add to that test, as before it showed the decision point (one result was `-20`, one was `-21`). I.e., can add
````
  printf("llvm_round_f32 %.1f\n", llvm_round_f32(-20.49)); // new line, should have different result
  printf("llvm_round_f32 %.1f\n", llvm_round_f32(-20.5));
  printf("llvm_round_f32 %.1f\n", llvm_round_f32(-20.51));
````

Reply:
Good idea, added.